ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,37 @@
 ==================================
-How to customize my catalog page 
+How To Customize Your Catalog Page 
 ==================================
 
 Product Catalog
 ===============
 
 All your published items show up in your catalog page (or *Shop* page).
 
-.. image:: ./media/shop.png
+.. image:: ./media/v14_product_catalog.png
```

Comment:
Unless this screenshot is very specific to V14, we avoid including the version tag in file names or in the content.